pipelines/run-pipelines-on-apache-airflow/README.md

```diff
@@ -0,0 +1,332 @@
+
+#### Add components from local sources
+
+To add component specifications to the registry that are stored locally:
+
+1. Open the **Component Catalogs** panel using one of the approaches mentioned above.
+
+   ![Add component registry entry](doc/images/add-component-catalog-entry.png)
+
+1. Add a new component registry entry by clicking `+` and selecting `New Filesystem Component Catalog`.
+   The first tutorial component you are adding to the registry makes an HTTP Request.
+1. Enter or select the following:
+   - **Name**: `request data`
+   - **Description**: `request data from GitHub API`
+   - **Category**: `request`
+   - **Runtime**: `APACHE_AIRFLOW`
+   - **Base Directory**: `.../examples/pipelines/run-pipelines-on-apache-airflow/components` (on Windows: `...\examples\pipelines\run-pipelines-on-apache-airflow\components`)
+   - **Paths**: `http_operator.py` 
+
+     > Note: Replace `...` with the path to the location where you cloned the Elyra example repository. The base directory can include `~` or `~user` to indicate the home directory. The concatenation of the base directory and each path must resolve to an absolute path or Elyra won't be able to locate the specified files.
+1. Save the component catalog entry.
+
+There are two approaches you can take to add multiple _related_ component specifications to the registry:
+- Specify multiple `Path` values.
+- Store the related specifications in the same directory and use the `Directory` **catalog type**. Elyra searches the directory for specifications. Check the **Include Subdirectories** checkbox to search subdirectories for component specifications as well.
+
+> Refer to the descriptions in the linked documentation topic for details and examples. 
+
+Locally stored component specifications have the advantage that they can be quickly loaded by Elyra. If you need to share component specifications with other users, ensure that the given **Paths** are the same relative paths across installations. The **Base Directory** can differ across installations.
+
+#### Add components from remote sources
+
+In version 3.3.0 Elyra's `URL Component Catalog` type only supports web resources that can be downloaded using HTTP `GET` requests, which don't require authentication.
+
+To add component specifications to the catalog that are stored remotely:
+
+1. Open the **Pipeline components** panel.
+1. Add a second component catalog entry, this time selecting `New URL Component Catalog` from the dropdown menu. This component executes a given bash command.
+1. Enter the following information:
+   - **Name**: `run command`
+   - **Description**: `run a shell script`
+   - **Category**: `scripting`
```

Comment:
'Runtime' is displayed before 'category' in the GUI. (Need to switch the order here)